### PR TITLE
Update contract docs

### PR DESCRIPTION
**Pull request by davidyuk**
related to #1256

### CODE REVIEW

File: docs/guides/contracts.md [hunk 1]
```diff
@@ -78,11 +78,10 @@ Note:
         - Key-value map with name of the include as key and source code of the include as value.
     - `validateByteCode` (default: false)
         - Compare source code with on-chain version.
-    - `opt` (default: {})
```
Comment (marc0olo) [inline docs/guides/contracts.md:81]: so `opt` not an explicit property anymore?

File: docs/guides/contracts.md [hunk 2]
```diff
@@ -78,11 +78,10 @@ Note:
...
-        - Object with other [transaction options](../transaction-options.md) which will be provided to **every transaction** that is initiated using the contract instance. You should be aware that:
-            - For most of these additional options it doesn't make sense to define them at contract instance level.
-            - You wouldn't want to provide an `amount` to each transaction or use the same `nonce` which would result in invalid transactions.
-            - For options like `ttl` or `gasPrice` it does absolutely make sense to set this on contract instance level.
+    - other [transaction options](../transaction-options.md) which will be provided to **every transaction** that is initiated using the contract instance. You should be aware that:
```
Comment (marc0olo) [inline docs/guides/contracts.md:81]: options just embedded as object, right? should maybe mentioned a bit more explicit here

Comment (marc0olo) [inline docs/guides/contracts.md:81]:
```suggestion
    - an object including other [transaction options](../transaction-options.md) which will be provided to **every transaction** that is initiated using the contract instance. You should be aware that:
```

Comment (davidyuk) [reply docs/guides/contracts.md:81]: there was an embedded object at `opt` field (`opt` inside `options`), but now it is inlined into `options` object.

Comment (davidyuk) [reply docs/guides/contracts.md:81]: there is no an object at the end of options, transactions options should be fields in `options` object

Comment (marc0olo) [reply docs/guides/contracts.md:81]: I might lack of javascript knowledge here. so having an `options` object as last param of a function call is usual and typically known to devs? I exactly meant what you wrote as answer. I just wanted to be explicit with the `options` object.

I edited the suggestion. is that better? or can we really leave like is?

**Review by mradkov — APPROVED** [review]

File: docs/guides/contracts.md (hunk 2 above)
Comment (mradkov) [reply docs/guides/contracts.md:81]: Looks good to me like this.

Comment (marc0olo) [reply docs/guides/contracts.md:81]: ahh you mentioned this in the call, right? I am fine with keeping it like is. no big deal :-)

Comment (davidyuk) [reply docs/guides/contracts.md:81]: > `options` object

is explained above, don't think necessary to repeat it 🙂

**Pull request merged**